src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessClientExtensions.cs

```diff
@@ -92,5 +93,27 @@ public static class ExceptionlessClientExtensions {
 
             return value;
         }
+
+        /// <summary>
+        /// SequenceToObjectValue
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="properties"></param>
+        /// <param name="key"></param>
+        /// <param name="defaultValue"></param>
+        /// <returns></returns>
+        internal static T SequenceToObjectValue<T>(this IReadOnlyList<LogEventProperty> properties,string key, T defaultValue)
+        {
+            var lProperty = properties.Where(a => a.Name.Equals(key, StringComparison.OrdinalIgnoreCase)).FirstOrDefault();
+
+            try
+            {
+                return (T)lProperty.Value.FlattenProperties();
```

Comment:
We shouldn't need a try catch here, if the value is null we should return the default. Is there any case I'm missing?